src/analytics/navigationTracker/navigationTracker.ts

```diff
@@ -171,22 +174,26 @@ export class NavigationTracker extends BaseTracker {
    * @param {HTMLLinkElement} element - The HTML link element to get the type of.
    * @return {string} The type of link: "footer", "header menu bar", "generic link", "generic button", or "undefined".
    */
-  getLinkType(element: HTMLLinkElement): string {
+  static getLinkType(element: HTMLLinkElement): string {
```

Comment:
Just like the form response tracker, is it worth using a switch statement here, seeing as there's embedded if statements?

Reply:
Good call yeah I've updated this one now 👍